python/test/t_InverseWishart_std.py

```diff
@@ -0,0 +1,171 @@
+#! /usr/bin/env python
+
+from __future__ import print_function
+import unittest as ut
+from math import pi, log, log10, floor
+
+import openturns as ot
+from openturns.testing import assert_almost_equal # (value, reference, rtol, atol)
+
+ot.TESTPREAMBLE()
+
+# Prevent from using unittest.TestLoader.loadTestsFromTestCase
+# so as not to get the test execution duration
+def loadTestsFromTestCase(cls):
+    """Launch the tests of the test case class"""
+    import sys
+    from inspect import getmembers
+    cls.setUpClass()
+    for member in getmembers(TestInverseWishartMethods):
+        member_name = member[0]
+        if member_name[0:5] == 'test_':
+            test = cls(member_name)
+            test.setUp()
+            print("Run " + member_name + "... ", end="")
+            sys.stdout.flush()
+            try:
+                test.debug()
+                print("SUCCESS")
+            except  Exception, exception:
+                print("FAILURE")
+                print(exception)
+            test.tearDown()
+    cls.tearDownClass()
```

Comment:
Unfortunately tearDownClass is new for 2.7, and we want to support python 2.6 a little longer,
would you mind not using it if version is < 2.7 ?
https://docs.python.org/2/library/unittest.html#unittest.TestCase.tearDownClass